src/controllers/JobRoleController.ts

```diff
@@ -0,0 +1,12 @@
+import Express from "express";
+import { getJobRoles } from "../services/JobRoleService"
+
+export const getAllJobRoles = async (req: Express.Request, res: Express.Response): Promise<void> => {
+    try{
+        res.render('pages/allJobRolesList.html', {jobRoles: await getJobRoles(), pageName: "Job Roles"});
```

Comment:
no need to pass the page name unless we're reusing the html

Reply:
The page name is used in the template HTML as we have it as a variable called {{ pageName }}

Reply:
fair enough